src/bridging-script/ios-bridge.js

```diff
@@ -0,0 +1,29 @@
+let MiniAppBridge = {}
+var queue = []
+
+MiniAppBridge.exec = function(action, onSuccess, onError) {
+    const callback = {}
+    callback.onSuccess = onSuccess;
+    callback.onError = onError;
+    callback.id = Math.random();
+    queue.unshift(callback)
+    webkit.messageHandlers.MiniApp.postMessage(JSON.stringify({action: action, id: callback.id}));
+}
+
+MiniAppBridge.execCallback = function(messageId, value) {
+    var queueObj = queue.find(callback => callback.id = messageId)
+    queueObj.onSuccess(value);
+    queue.shift(queueObj)
```

Comment:
I think this isn't correct, is it? `shift` can only remove the first element, it doesn't accept parameters: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/shift

I think this needs to use `slice` maybe?

Reply:
Looks like slice returns a new array object, so I have used `splice()`